packages/bvaughn-architecture-demo/components/inspector/KeyValueRenderer.tsx

```diff
@@ -151,7 +151,7 @@ export default function KeyValueRenderer({
       {before}
       {name != null ? (
         <>
-          <span className={nameClass}>{name}</span>
+          <span className={nameClass} data-test-name="Key">{name}</span>
```

Comment:
This test name feels a bit too generic. Could we do "KeyValue-Header" or something?